bluemarket/src/index.js

```diff
@@ -1,8 +1,10 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
+import App from './App.jsx';
 import * as serviceWorker from './serviceWorker';
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
Once I can figure out where the import statements are coming from, I should be able to understand the code more.

Reply:
the imports starting with ./ are coming from the src folder at the specified location (ex ./App.jsx is the file App.jsx in the src folder), and the ones that start with the name (like `import 'bootstrap/dist/css/bootstrap.min.css' are coming from the node_modules folder. The latter ones are not worth fully understanding, I just saw that's how to get the style online and copied it. The node_modules folder is just us referencing other people stuff.
`